src/components/Groups.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export default function Groups({ groupsChosen }) {
+  const renderGroup = groupsChosen.map((group) => group.join(", "));
+  return (
+    <div className="valuesToChoose">
+      <h2>👬👭 {groupsChosen.length} Groups 👬👭</h2>
+      <ul className="RealDecide">
```

Comment:
I would rename it to camelCase like other classNames
```suggestion
      <ul className="realDecide">
```